app/controllers/user_languages_controller.rb

```diff
@@ -1,19 +1,53 @@
-# class UserLanguagesController < ApplicationController
-#   def options
-#     @user_languages = UserLanguage.find()
-#   end
+class UserLanguagesController < ApplicationController
+  def new_language #new
+    @user_language = UserLanguage.new
+    @all_languages = Language.all
+    capitalize(@all_languages)
+  end
 
-#   # def update_options
-#   #   @user_language =
-#   # end
-#   # code after merging column w Esh
+  #creats languages fine but errors do not render properly
+  def create_language #create
+    @name = params[:user_language][:language].reject(&:blank?) # @name is an array of languages user inputs
+    if @name.blank?
+      flash[:error] = "You must select at least one language"
+      render :new_language
```

Comment:
errors do not render and flash message doesnt appear!
However validations work very well